src/clandinin_lab_to_nwb/brezovec/brezovecimagingextractor.py

```diff
@@ -176,7 +176,7 @@ def get_video(
         end_frame = end_frame or self.get_num_frames()
         start_frame = start_frame or 0
 
-        return self.nibabel_image.dataobj[:, :, :, start_frame:end_frame]
+        return self.nibabel_image.dataobj[:, :, :, start_frame:end_frame].transpose(3, 0, 1, 2)
```

Comment:
@alessandratrapani this is the thing that was missing.
We have the convention of returning time, rows, colums and planes. We should document this probably on the abstract method.

Reply:
For clarification on the eternal confusion - for an `ImagingExtractor` or other images data objects in ROI Extractors

'rows' (as a matrix) := 'height' or 'y' (in the usual image convention)
'columns (as a matrix) := 'width' or 'x'

Whereas in NWB this aspect is transposed as (width x height)

Right? Is that accounted for here?

Reply:
Yes, this is something that we doubled checked. Thanks for the reminder!

Reply:
We now pointed this out explicitly in the notes and in the `get_video` function.